accessibility-101.md

```diff
@@ -0,0 +1,196 @@
+
+This is one for the final category I just listed. You may think that accessibility doesn't apply to you or to your users, so why implement it? What would a blind person do with a photo editing tool?
+
+The truth is, you're right to a certain degree. If you have done meticulous user research and have excluded any chance of a certain group of people visiting your website, the priority for catering to that group of people diminishes quite a bit. 
+
+However, doing this research is key in actually defending such a statement. Did you know there were [blind gamers?](http://audiogames.net) and even [blind photographers?](http://peteeckert.com/). Perhaps you knew [musicians can be deaf](http://mentalfloss.com/article/25750/roll-over-beethoven-6-modern-deaf-musicians)?
+
+If you did, good for you. If not, I guess this drives my point home all the more. :)
+
+The picture gets even more complicated when we look at legislation that actually forces you to make certain websites and web apps accessible. 
+A prime example is the US-based [section 508](http://jimthatcher.com/webcourse1.htm). Right now, this law mainly refers to government organisations, public sector websites etc. However, laws change. 
+
+Last year, airline websites were included in this list which meant that even here in Europe, airline website devs scrambled to make their content accessible. Not doing so can get your company a fine of literally tens of thousands of dollars for each day the problem isn't fixed.
+
+There's variations on this legislation all over the world, some more severe and all-encompassing than others. Not knowing about that fact doesn't make the lawsuit go away, sadly. :)
+
+### Ok, so accessibility is a big deal. Now how do we implement  it?
+
+That question, sadly, is harder to answer than it may seem. The Harry Potter quote at the top is there for a reason, and its not my being an avid Fantasy reader. :)
+
+As I stated above, accessibility is important for a large group of different people, each with their own needs. Making your website work for literally everyone is a large, on-going task.
+
+To bring a bit of a method to the madness, the Web Content Accessibility Guidelines or [WCAG](https://www.wuhcag.com/web-content-accessibility-guidelines/) were composed. This document contains a number of criteria you can use to check your website.
+For now, I will cover some of the most important basics here. I will point you at the low-hanging fruits, so to speak. In subsequent articles, I will discuss more advanced techniques like [WAI-ARIA]  which is important for JavaScript-based apps.
+
+#### Talk like the natives
+The HTML specification is a document that describes how the language should be used to build websites. Assistive technologies, like screenreaders, speech recognition programs etc. are aware of this document. Web developers however, often are not, or at least not enough, and think something like this is ok:
+
+```html
+<div class="awesome-button"</div>
```

Comment:
Hmm. Good catches, but the page is live now. Would I have to make a
new branch, fix these issues and submit another PR, or is there a
quicker way to fix hese?